library/common/extensions/filters/http/platform_bridge/filter.cc

```diff
@@ -504,6 +514,18 @@ void PlatformBridgeFilter::resumeEncoding() {
   });
 }
 
+void PlatformBridgeFilter::resetIdleTimeout() {
+  ENVOY_LOG(trace, "PlatformBridgeFilter({})::resumeEncoding", filter_name_);
+
+  auto weak_self = weak_from_this();
+  dispatcher_.post([weak_self]() -> void {
+    if (auto self = weak_self.lock()) {
+      // Stream idle timeout is nondirectional.
+      //self->decoder_callbacks_->resetStreamIdleTimeout();
```

Comment:
```suggestion
      //decoder_callbacks_->resetStreamIdleTimeout();
```

Reply:
Accessing through `self` is strictly necessary here, I think.